MonoGame.Framework.Content.Pipeline/EffectImporter.cs

```diff
@@ -28,9 +29,24 @@ public EffectImporter()
         /// <returns>Resulting game asset.</returns>
         public override EffectContent Import(string filename, ContentImporterContext context)
         {
+            using (var stream = new FileStream(filename, FileMode.Open, FileAccess.Read))
+            {
+                return Import(stream, filename, context);
+            }
+        }
+
+        /// <summary>
+        /// Called by third-party systems when importing an .fx file to be used as a game asset.
```

Comment:
Leave out the comment about third-party systems.  It should simply be something like "Imports an effect file from a stream."  The <returns> should be "The effect content."